cobie-excel-connector/package.json

```diff
@@ -16,9 +16,7 @@
     "@bentley/spatial-composition-schema": "1.0.0",
     "@itwin/core-backend": "^3.0.0",
     "@itwin/ecschema-metadata": "^3.0.0",
-    "@itwin/imodels-access-backend": "^1.0.2",
```

Comment:
We can remove imodels-access-backend and imodels-client-authoring b/c they are pulled through pcf w/ the new version 0.3.0-2 correct?